packages/cli-kit/src/public/node/output.ts

```diff
@@ -320,11 +332,25 @@ export function outputWarn(content: OutputMessage, logger: Logger = consoleWarn)
   outputWhereAppropriate('warn', logger, message)
 }
 
+/**
+ * Logs an unformatted message at the given log level.
+ * Note: By default,  messages are sent through the standard error.
+ *
+ * @param content - The content to be output to the user.
+ * @param logLevel - The log level associated with the message.
+ * @param logger - The logging function to use to output to the user.
+ */
+export function output(content: OutputMessage, logLevel: LogLevel = 'info', logger: Logger = consoleWarn): void {
+  if (isUnitTest()) collectLog(logLevel, content)
+  const message = stringifyMessage(content)
+  outputWhereAppropriate(logLevel, logger, message)
+}
+
 /**
  * Prints a new line in the terminal.
  */
 export function outputNewline(): void {
-  console.log()
+  console.warn()
```

Comment:
`console.warn` should be avoided, as it may decorate output text. Instead, we should reuse `consoleWarn` which doesn't use `console` at all.